tests/foreman/ui/test_discoveredhosts.py

```diff
@@ -99,9 +168,16 @@ def test_delete_discovered_host_1(self):
 
         @Assert: Selected host should be removed successfully
 
-        @Status: Manual
-
         """
+        mac = gen_mac(multicast=True, locally=True)
+        hostname = 'mac{0}'.format(mac.replace(':', ""))
+        self._pxe_boot_host(mac)
+        with Session(self.browser) as session:
+            session.nav.go_to_select_org(self.org_name)
+            self.navigator.go_to_discovered_hosts()
+            self.assertIsNotNone(self.discoveredhosts.search(hostname))
```

Comment:
To be honest, I thought on your words a lot and it seems as really good practice to have assertion on a test-level.
But here, it is almost impossible to implement logic like this without *huge* additional amount of code:
- Adding `timeout` parameter into every search method (as it was implemented in some methods before) will not work as you will never know what number of seconds will be enough (1 or 3 or 10). Putting here big enough number will back us to bad performance once more. And anyway, in result we will have a lot of redundant code and a lot of static timeout constants which is much worse practice than having logic on a wrong level
or
- Adding code like:
```
searched = self.search_entity(
                name,
                name_locator,
                search_key=search_key,
                button_timeout=3,
                result_timeout=1,
            )
```
per each entity also seems as pretty insane idea.
So, I decided to perform implementation like in #2809